src/systemd/sd-dhcp-client.h

```diff
@@ -202,6 +202,8 @@ sd_dhcp_client *sd_dhcp_client_unref(sd_dhcp_client *client);
  * options when using RFC7844 Anonymity Profiles */
 int sd_dhcp_client_new(sd_dhcp_client **ret, int anonymize);
 
+int sd_dhcp_client_id_to_string(char **ret, const void *data, size_t len);
```

Comment:
`ret` should be the final argument here. (The output parameter is the first parameter only for "constructor" functions, an not when just returning a string.)

Reply:
ok, thanks